cpp/references/data_structures/list_digraph.cpp

```diff
@@ -1 +1,596 @@
-;
\ No newline at end of file
+#include "list_digraph.hpp"
+
+using namespace godot;
+
+typedef lemon::ListDigraph::Arc LemonArc;
```

Comment:
Move these typedefs into `list_digraph.hpp` so you can use them in definitions there too.